tests/core/device-reg/task.yaml

```
@@ -7,7 +7,7 @@ execute: |
     . "$TESTSLIB"/names.sh
 
     echo "Wait for first boot to be done"
-    while ! snap changes | grep -q "Done.*Initialize system state"; do sleep 1; done
+    retry -n 200 --wait 1 sh -c 'snap changes | MATCH "Done.*Initialize device"'
```

Oh that's not good indeed. We will fail faster now if something is wrong which is good 👍

---

done